dracut/99setup-root/initrd-setup-root

```diff
@@ -153,6 +153,6 @@ mkdir -p /sysroot/.etc-work
 mount -t overlay overlay -o lowerdir=/sysroot/usr/share/flatcar/etc,upperdir=/sysroot/etc,workdir=/sysroot/.etc-work,redirect_dir=on,metacopy=off,noatime /sysroot/etc
 
 # PXE initrds may provide OEM
-if [ -d /usr/share/oem ] && mountpoint --quiet /sysroot/usr/share/oem; then
-    cp -Ra /usr/share/oem/. /sysroot/usr/share/oem
+if [ -d /oem ] && mountpoint --quiet /sysroot/oem; then
+    cp -Ra /oem/. /sysroot/oem
```

Comment:
This will break existing custom initrds - not sure if that is used but since we don't know, it needs to copy from `/usr/share/oem/.`

Reply:
Reverted the change.